cmd/account/console.go

```diff
@@ -104,14 +105,22 @@ func (o *consoleOptions) run() error {
 
 	ctx := context.TODO()
 	var accountID string
+	var isBYOC bool
+	var acctSuffix string
 	if o.accountName != "" {
 		account, err := k8s.GetAWSAccount(ctx, o.kubeCli, o.accountNamespace, o.accountName)
 		if err != nil {
 			return err
 		}
 		accountID = account.Spec.AwsAccountID
+		isBYOC = account.Spec.BYOC
+		if isBYOC {
+			acctSuffixSlice := strings.Split(o.accountName, "osd-creds-mgmt-")
```

Comment:
This should be as a label on the Account CR, and IMO it should be considered the source of truth

Reply:
Oooh, that's so much easier!  Thanks!

Something something monday.

Reply:
You can look at #66 for this as well :)